src/SourceTerm/typedefs.hpp

```diff
@@ -75,11 +75,11 @@ namespace seissol {
        * FSRM: Moment tensor */
       real (*tensor)[TensorSize];
 
-      /// mu*Area
-      real *muA;
+      /// Area
+      real *A;
 
-      /// lambda*Area
-      real *lambdaA;
+      /// elasticity tensor
+      real (*cij)[81];
```

Comment:
std::array/std::vector would be better here to avoid the hassle of manual memory management